.github/CODEOWNERS

```diff
@@ -5,3 +5,4 @@
 /packages/payment-integration @bigcommerce/checkout
 /packages/payment-integration-test-utils @bigcommerce/checkout
 /packages/paypal-commerce @bigcommerce/kyiv-payments-team
+/packages/squarev2-integration @bigcommerce/apex-leads
```

Comment:
Please set this to @bigcommerce/apex-integrations

Reply:
<img src="https://media0.giphy.com/media/jx5MJyZqAFsLS/giphy.gif"/>

Reply:
Done.